user/mongoStoreClient.go

```diff
@@ -125,6 +122,22 @@ func (d MongoStoreClient) FindUsers(user *User) (results []*User, err error) {
 	return results, nil
 }
 
+func (d MongoStoreClient) FindUsersByRole(role string) (results []*User, err error) {
+	cpy := d.session.Copy()
+	defer cpy.Close()
+
+	if err = mgoUsersCollection(cpy).Find(bson.M{"roles": role}).All(&results); err != nil {
```

Comment:
The older API allowed you to specify multiple roles to search on and if the user had any one of those roles it would match.  (So the "in" here was does the user have any roles *in* that `user.Roles` list.)  The new code is a shortcut when you only have one item you are looking for in the array you can do what I have there.

See: https://docs.mongodb.com/manual/tutorial/query-arrays/#query-an-array-for-an-element